app/controllers/serials_controller.rb

```diff
@@ -1,6 +1,8 @@
 class SerialsController < ApplicationController
   def index
     @serials = current_site.serials.published.order(id: :desc).page(params[:page])
```

Comment:
By the way, it is not related to this pull request, but what is `order(id: :desc)`? I don't know its intention from this code.